client/webserver/site/src/js/doc.js

```diff
@@ -14,13 +14,12 @@ const BipIDs = {
 
 const BipSymbols = Object.values(BipIDs)
 
-// Parameters for printing asset values.
-const fullPrecisionSpecs = {
-  minimumSignificantDigits: 4,
-  maximumSignificantDigits: 8,
-  minimumFractionDigits: 8,
+const intFormatter = new Intl.NumberFormat(navigator.languages)
+
+const decimalFormatter = new Intl.NumberFormat(navigator.languages, {
```

Comment:
`navigator.languages` should maybe come from a cookie based on the dexc config setting instead.